components/raftstore/src/store/peer_storage.rs

```diff
@@ -158,9 +158,20 @@ impl EntryCache {
             let first_index = entries[0].get_index();
             if cache_last_index >= first_index {
                 if self.cache.front().unwrap().get_index() >= first_index {
+                    let mut cleared_cache_entries_size = 0;
+                    self.cache
+                        .iter()
+                        .for_each(|e| cleared_cache_entries_size += e.compute_size());
```

Comment:
This is wrong. `compute_size` is the serialized size instead of memory size. You should use `std::mem::size_of_val(&e) + e.get_data().len() + e.get_context().len()`.

Reply:
I think it's difficult to use capacity to statistics memory size.

Reply:
Fixed. PTAL~

Reply:
@BusyJay  @zhangjinpeng1987  Fixed.  PTAL~      
Thanks!